pages/api/manifest.ts

```diff
@@ -6,7 +6,7 @@ import { toNextHandler } from "retes/adapter";
 import { Response } from "retes/response";
 
 import * as GeneratedGraphQL from "../../generated/graphql";
-import { name, version } from "../../package.json";
+import packageInfo from "../../package.json";
```

Comment:
suggestion: maybe just `packageJson`? 

Also, why there was warning? It doesnt look like it changes anything?

Reply:
Yup, that's the better name.